R-package/README.md

```diff
@@ -0,0 +1,564 @@
+
+<br>
+
+Before the installation of *rgf* I might have to remove *Rtools* environment variables, such as **C:\\Rtools\\bin** (accessible as explained previously), otherwise errors might occur.
+
+<br>
+
+**Installation of RGF, FastRGF and rgf_python**   [ *assuming the installation takes place in the **c:/** directory* ]
+
+<br>
+
+Open a console with **administrator privileges** (right click on cmd and *run as administrator*), then 
+<br>
+
+
+```R
+
+
+# download the most recent version of rgf-python from the GitHub repository 
+#--------------------------------------------------------------------------
+
+git clone http://github.com/RGF-team/rgf.git
+
+
+
+# then navigate to 
+#-----------------
+
+cd /rgf/RGF/
+
+mkdir bin
+
+cd c:/
+
+
+# then download the latest "rgf.exe" from https://github.com/RGF-team/rgf/releases and place the "rgf.exe" inside the previously created "bin" folder ( /rgf/RGF/bin )
+
+
+
+# installation of RGF
+#--------------------
+
+cd /rgf/RGF/build
+
+mingw32-make
+
+cd c:/
+
+
+
+# installation of FastRGF
+#------------------------
+
+cd /rgf/FastRGF/
+
+mkdir build
+
+cd build
+
+
+# BEFORE PROCEEDING WITH cmake MAKE SURE THAT THE "Rtools" folder IS NOT IN THE SAME DIRECTORY (IF THAT IS THE CASE THEN REMOVE IT TEMPROARILY, i.e. copy-paste the "Rtools" folder somewhere else)
+
+
+cmake .. -G "MinGW Makefiles"
+
+mingw32-make
+
+mingw32-make install
+
+cd c:/
+
+
+# IF APPLICABLE, PASTE THE "Rtools" FOLDER IN THE INITIAL LOCATION / DIRECTORY
+
+
+# installation of rgf-python
+#---------------------------
+
+cd rgf/python-package
+
+python setup.py install
+
+
+```
+
+<br>
+
+Then open a command prompt (console) and type,
+
+<br>
+
+```R
+
+python 
+
+
+```
+
+<br>
+
+to launch *Python* and then type 
+
+<br>
+
+```R
+
+import rgf
+
+exit()
+
+```
+
+<br>
+
+to observe if *rgf* is installed properly. Then continue with the installation of the RGF package,
+
+<br>
+
+```R
+
+install.packages(RGF)
+
+
+```
+
+<br>
+
+On windows the user can take advantage of the RGF package currently **only** from within the command prompt (console). First, find the full path of the installation location of R (possible if someone right-clicks in the R short-cut (probably on Desktop) and navigates to properties >> shortcut >> target). In case, for instance, that R is located in **C:\\Program Files\\R\\R-3.4.0\\bin\\x64\\R**, then, by opening a command prompt (console) and giving,
+
+<br>
+
+```R
+
+cd C:\Program Files\R\R-3.4.0\bin\x64\
+
+R
+
+library(RGF)
+
+
+```
+<br>
+
+one can proceed with the usage of the RGF package.
+
+<br>
+
+
+### **Installation of the RGF package**
+
+<br>
+
+To install the package from CRAN use, 
+
+<br>
+
+```R
+
+install.packages('RGF')
+
+
+```
+<br>
+
+and to download the latest version from GitHub use the *install_github* function of the devtools package,
+<br><br>
+
+```R
+
+devtools::install_github(repo = 'mlampros/RGF')
```

Comment:
Update command

Reply:
can you please elaborate on which the web link address will be because the R package does not currently exist.

Reply:
Sure:
```
devtools::install_github(repo = 'RGF-team/rgf', subdir = 'R-package')
```

Reply:
thanks.